charts/images.yaml

```diff
@@ -68,32 +68,56 @@ images:
 - name: csi-provisioner
   sourceRepository: github.com/kubernetes-csi/external-provisioner
   repository: k8s.gcr.io/sig-storage/csi-provisioner
-  tag: "v2.1.2"
+  tag: "v2.2.2"
+  targetVersion: "< 1.20"
+- name: csi-provisioner
+  sourceRepository: github.com/kubernetes-csi/external-provisioner
+  repository: k8s.gcr.io/sig-storage/csi-provisioner
+  tag: "v3.2.0"
+  targetVersion: ">= 1.20"
 - name: csi-attacher
   sourceRepository: github.com/kubernetes-csi/external-attacher
   repository: k8s.gcr.io/sig-storage/csi-attacher
-  tag: "v3.3.0"
+  tag: "v3.4.0"
+- name: csi-resizer
+  sourceRepository: github.com/kubernetes-csi/external-resizer
+  repository: k8s.gcr.io/sig-storage/csi-resizer
+  tag: "v1.5.0"
```

Comment:
Ref https://github.com/kubernetes-csi/external-resizer/blob/master/CHANGELOG/CHANGELOG-1.0.md:
> [...] New RBAC rules for listing pods are required. [...]

You need to add the following RBAC rules:
```diff
diff --git a/charts/internal/shoot-system-components/charts/csi-driver-node/templates/clusterrole-csi-resizer.yaml b/charts/internal/shoot-system-components/charts/csi-driver-node/templates/clusterrole-csi-resizer.yaml
index f11ccfa6..902ae3ab 100644
--- a/charts/internal/shoot-system-components/charts/csi-driver-node/templates/clusterrole-csi-resizer.yaml
+++ b/charts/internal/shoot-system-components/charts/csi-driver-node/templates/clusterrole-csi-resizer.yaml
@@ -10,6 +10,9 @@ rules:
 - apiGroups: [""]
   resources: ["persistentvolumeclaims"]
   verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
 - apiGroups: [""]
   resources: ["persistentvolumeclaims/status"]
   verbs: ["update", "patch"]
```

Otherwise resize of PVC fails:
```
W0623 14:16:57.670414       1 reflector.go:324] k8s.io/client-go/informers/factory.go:134: failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:kube-system:csi-resizer" cannot list resource "pods" in API group "" at the cluster scope
E0623 14:16:57.670443       1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Pod: failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:kube-system:csi-resizer" cannot list resource "pods" in API group "" at the cluster scope
```

Reply:
I now read:
> This behaviour is enabled by default but can be disabled by setting handle-volume-inuse-error to false, which would allow external-resizer to perform volume expansion without checking if volume being expanded is in-use or not(https://github.com/kubernetes-csi/external-resizer/pull/89, [@saikat-royc](https://github.com/saikat-royc)). If CSI driver being used supports online expansion, it might be desirable to set handle-volume-inuse-error to false - to save costs associated with watching all pods in the cluster.

I see that aws-ebs-csi-driver charts already set `handle-volume-inuse-error=false` - ref https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1280. I assume that online expansion is supported by the aws-ebs-csi-driver. 

Then instead of allowing watch for Pods let's rather have:
```diff
diff --git a/charts/internal/seed-controlplane/charts/csi-driver-controller/templates/csi-driver-controller-deployment.yaml b/charts/internal/seed-controlplane/charts/csi-driver-controller/templates/csi-driver-controller-deployment.yaml
index e366e12a..bc343ccd 100644
--- a/charts/internal/seed-controlplane/charts/csi-driver-controller/templates/csi-driver-controller-deployment.yaml
+++ b/charts/internal/seed-controlplane/charts/csi-driver-controller/templates/csi-driver-controller-deployment.yaml
@@ -160,6 +160,7 @@ spec:
         - --kubeconfig=/var/run/secrets/gardener.cloud/shoot/generic-kubeconfig/kubeconfig
         - --leader-election=true
         - --leader-election-namespace=kube-system
+        - --handle-volume-inuse-error=false
         - --v=5
         env:
         - name: ADDRESS
```